api.proto

```diff
@@ -79,7 +79,8 @@ message SubscribeRequest {
     StartPosition startPosition  = 3; // Where to begin consuming from
     int64         startOffset    = 4 [jstype=JS_STRING]; // Offset to begin consuming from
     int64         startTimestamp = 5 [jstype=JS_STRING]; // Timestamp to begin consuming from
-    bool          readISRReplica    = 6;  // Subscribe from a random ISR replica instead of choosing explicitly leader
+    bool          readISRReplica = 6; // Subscribe from a random ISR replica instead of choosing explicitly leader
+    bool          allowResuming  = 7; // Allow resuming a paused partition before subscribing to it
```

Comment:
I'd just call it `resume` or `resumePaused`. I think `allowResuming` is a bit confusing.

Reply:
Hm yeah I agree. I have made the change.